documentation/offline-site.md

```diff
@@ -48,18 +48,4 @@ RELEASE=3.11.0
 - When the change is merged, the new version will be synchronised
 
 ## How to test
```

Comment:
I suggest removign the `## How to test` section

Reply:
removed